build.sh

```diff
@@ -4,19 +4,33 @@
 git submodule update --init
 
 # Set environment variables
-cd cedar-drt && source ./set_env_vars.sh
+cd cedar-drt
+source set_env_vars.sh
+cd ..
+
+# Build the Dafny formalization and extract to Java code
```

Comment:
Nit: should we create a flag to enable separate installation of Dafny or Lean?

Reply:
I don't think it's necessary since we don't expect long-term support for Dafny